timeless/uploads/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""File Uploads"""
+from flask_uploads import UploadSet, IMAGES as IMAGES_, configure_uploads
+
+
+IMAGES = None
+
+
+def images(app):
+    """
+    Creates images upload set.
+    :param app: Flask application instance
+    :return: Images upload set created
+    @todo #205:30min Introduce Flask-Uploads configuration parameters
+     to app config.According to Flask-Uploads, there is a need to introduce
+     additional configuration parameters to the application config instance.
+     Make sure UPLOADS_DEFAULT_URL, UPLOADED_IMAGES_URL values are calculated
+     according to the current runtime host/port information and
+     also UPLOADS_DEFAULT_DEST and UPLOADED_IMAGES_DEST values
+     point to correct static resources path
+    """
+    # Configure the image uploading via Flask-Uploads
+    result = UploadSet("images", IMAGES_)
+    path = "/project/static/img/"
+    app.config["UPLOADS_DEFAULT_DEST"] = app.instance_path + path
+    app.config["UPLOADS_DEFAULT_URL"] = "http://localhost:5000/static/img/"
```

Comment:
@ekondrashev The url should be in the config file (or use one from the config file)

Reply:
@ammaratef45 originally I was about adding those to the `config` but if you take a closer look, the value for those parametrs should be calculated dynamically based on current address and also static folder location which we can take from the app instance only. Do I miss anything?

Reply:
@ekondrashev I'm not complaining about `app.instance_path` but about `"http://localhost:5000/static/img/"`
Those hardcoded urls should be avoided, right?

Reply:
@ammaratef45 exactly, but there is no way to avoid it and both provide correct value at this moment, so there is a puzzle to address this above